packages/ecdsa/src/main.nr

```diff
@@ -0,0 +1,12 @@
+mod lib;
+
+fn main(private_key: [u8; 32], message: [u8; 32], random_nonce: [u8; 32]) {
```

Comment:
Can we have a test and a proper README with instructions on how to use, import, test, and with a benchmark? Seems like this library would benefit from using [noir-edwards](https://github.com/noir-lang/noir-edwards/tree/main) too.

A lot of this code is AI-generated.

Reply:
On it. 

Yes, AI Generated.

Reply:
Optimised version of BabyJubJub already implements noir-edwards I believe.